src/main/java/de/dagere/peass/ci/logs/rts/AggregatedRTSResult.java

```diff
@@ -3,29 +3,37 @@
 import de.dagere.peass.ci.RTSResult;
 
 public class AggregatedRTSResult {
-   private final RTSLogSummary logSummary;
```

Comment:
Please do not reform the code, since this will create a lot of boilerplate changes in the version history